src/TagInput/driver.js

```diff
@@ -0,0 +1,25 @@
+export default class TagInputDriver
+{
+    constructor( wrapper )
+    {
+        this.wrapper = wrapper;
+    }
+
+    clickClose()
```

Comment:
This method should make use of the `clickClose` method of the Tag component’s own driver